src/flint/types/fmpz_poly.pyx

```diff
@@ -648,3 +730,14 @@ cdef class fmpz_poly(flint_poly):
                     return int(i)
         libc.stdlib.free(phi)
         return 0
+
+    def content(self):
+        """
+        Return the GCD of the coefficients of ``self``.
+
+            >>> fmpz_poly([3, 6, 0]).content()
+            3
+        """
+        cdef fmpz res = fmpz()
+        _fmpz_vec_content(res.val, self.val.coeffs, self.val.length)
+        return res
```

Comment:
We may need to think a bit about the sign convention for `primitive` and `content`. Typically the primitive part is defined to be a canonical representative in some sense which may mean that the content should absorb the sign although there are different ways of defining this.